cloud/deploy/deploy.go

```diff
@@ -154,7 +166,7 @@ func Deploy(path, runtimeID, wsID, pytest, envFile, imageName, deploymentName st
 		domain = splitDomain[1]
```

Comment:
Is there a reason why we are not using `c.Domain` directly as it does not have the `cloud` subdomain in it.

Reply:
I dont think we need this anymore since we have updated code to always save the context without the cloud subdomain.

@sunkickr Do you see any reason why we might need it?